libraries/Microsoft.Bot.Builder.AI.QnA/Models/AnswerSpanResponse.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    public class AnswerSpanResponse
+    {
+        /// <summary>
+        /// Gets or sets the answer text.
```

Comment:
The quality of all of these doc comments need significant improvement.

Reply:
Done.

---
In reply to: [430552481](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430552481) [](ancestors = 430552481)